packages/block-library/src/playlist/edit.js

```diff
@@ -0,0 +1,469 @@
+				order: trackOrder,
+				currentTrack:
+					sortedTracks.length > 0 &&
+					sortedTracks[ 0 ].uniqueId !== currentTrack
+						? sortedTracks[ 0 ].uniqueId
+						: currentTrack,
+			} );
+		},
+		[
+			clientId,
+			currentTrack,
+			innerBlockTracks,
+			replaceInnerBlocks,
+			setAttributes,
+		]
+	);
+
+	function toggleAttribute( attribute ) {
+		return ( newValue ) => {
+			setAttributes( { [ attribute ]: newValue } );
+		};
+	}
+
+	const hasSelectedChild = useSelect(
+		( select ) =>
+			select( blockEditorStore ).hasSelectedInnerBlock( clientId ),
+		[ clientId ]
+	);
+
+	const hasAnySelected = isSelected || hasSelectedChild;
+
+	const innerBlocksProps = useInnerBlocksProps( blockProps, {
+		__experimentalAppenderTagName: 'li',
+		renderAppender: hasAnySelected && InnerBlocks.ButtonBlockAppender,
+	} );
+
+	if ( ! tracks || ( Array.isArray( tracks ) && tracks.length === 0 ) ) {
+		return (
+			<div
+				{ ...blockProps }
+				className={ clsx( 'is-placeholder', blockProps.className ) }
+			>
+				<MediaPlaceholder
+					icon={ <BlockIcon icon={ icon } /> }
+					labels={ {
+						title: __( 'Playlist' ),
+						instructions: __(
+							'Upload an audio file or pick one from your media library.'
+						),
+					} }
+					onSelect={ onSelectTracks }
+					accept="audio/*"
+					multiple
+					allowedTypes={ ALLOWED_MEDIA_TYPES }
+					onError={ onUploadError }
+				/>
+			</div>
+		);
+	}
+
+	return (
+		<>
+			<BlockControls group="other">
+				<MediaReplaceFlow
+					name={ __( 'Edit' ) }
+					onSelect={ onSelectTracks }
+					accept="audio/*"
+					multiple
+					mediaIds={ tracks
+						.filter( ( track ) => track.id )
+						.map( ( track ) => track.id ) }
+					allowedTypes={ ALLOWED_MEDIA_TYPES }
+					onError={ onUploadError }
+				/>
+			</BlockControls>
+			<InspectorControls>
+				<PanelBody title={ __( 'Settings' ) }>
+					<ToggleControl
+						__nextHasNoMarginBottom
+						label={ __( 'Show Tracklist' ) }
+						onChange={ toggleAttribute( 'showTracklist' ) }
+						checked={ showTracklist }
+					/>
+					{ showTracklist && (
+						<>
+							<ToggleControl
+								__nextHasNoMarginBottom
+								label={ __( 'Show artist name in Tracklist' ) }
+								onChange={ toggleAttribute( 'showArtists' ) }
+								checked={ showArtists }
+							/>
+							<ToggleControl
+								__nextHasNoMarginBottom
+								label={ __( 'Show number in Tracklist' ) }
+								onChange={ toggleAttribute( 'showNumbers' ) }
+								checked={ showNumbers }
+							/>
+						</>
+					) }
+					<ToggleControl
+						__nextHasNoMarginBottom
+						label={ __( 'Show images' ) }
+						onChange={ toggleAttribute( 'showImages' ) }
+						checked={ showImages }
+					/>
+					<SelectControl
+						__next40pxDefaultSize
+						__nextHasNoMarginBottom
+						label={ __( 'Order' ) }
+						value={ order }
+						options={ [
+							{ label: __( 'Descending' ), value: 'DESC' },
+							{ label: __( 'Ascending' ), value: 'ASC' },
```

Comment:
```suggestion
							{ label: __( 'Descending' ), value: 'desc' },
							{ label: __( 'Ascending' ), value: 'asc' },
```

In general, I think it's best to use lowercase values.